StudyPathTraker/StudyPathTraker/Collection/SPCategoriesCollectionViewFlowDelegate.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  SPCategoriesCollectionViewFlowDelegate.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 4/30/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import UIKit
+
+
+class SPCategoriesCollectionViewFlowDelegate: NSObject, UICollectionViewDelegateFlowLayout {
+    
+    private let categoryEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
```

Comment:
this would be better if you inject as a configuration object, same for the other constants that you have.

Reply:
ready